.github/CONTRIBUTING.md

```diff
@@ -9,6 +9,7 @@ In this document, you'll learn how to run Briefer in development mode and how to
 - [Running Briefer in dev mode](#computer-running-briefer-in-dev-mode)
 - [Questions, bugs, and feature requests](#bulb-questions-bugs-and-feature-requests)
 - [Submitting pull requests](#repeat-submitting-pull-requests)
+- [Possible Error fixes](#warning-possible-error-fixes)
```

Comment:
I think we should call this section `Troubleshooting` and adjust the link accordingly.

Btw, I think a better emoji would be 🔧  - I think a "warning" draws too much attention here.